Jellyfin.Api/Constants/Policies.cs

```diff
@@ -45,6 +45,11 @@ public static class Policies
         /// </summary>
         public const string LocalAccessOrRequiresElevation = "LocalAccessOrRequiresElevation";
 
+        /// <summary>
+        /// Policy name for requiring local LAN access.
+        /// </summary>
+        public const string NetworkAccessPolicy = "NetworkAccessPolicy";
```

Comment:
```suggestion
        public const string LocalNetworkAccessPolicy = "LocalNetworkAccessPolicy";
```

Should change the name to be explicit in that the policy is only for "Local" access